Source/displayinfo/include/ExtendedDisplayIdentification.h

```diff
@@ -303,31 +717,147 @@ namespace Plugin {
+                        colorFormatMap |= static_cast<uint8_t>(colorformattype::YCBCR_4_2_2);
+                        break;
+                    case 0x03:
+                        colorFormatMap |= static_cast<uint8_t>(colorformattype::RGB);
+                        colorFormatMap |= static_cast<uint8_t>(colorformattype::YCBCR_4_2_2);
+                        colorFormatMap |= static_cast<uint8_t>(colorformattype::YCBCR_4_4_4);
+                        break;
+                }
+            }
+            return colorFormatMap;
+        }
+
+        uint8_t SupportedColorFormats() const {
+            uint8_t colorFormatMap = 0;
+
+            Iterator segment = Iterator(_segments);
+            while(segment.Next() == true) {
+                if(segment.Type() == CEA::extension_tag) {
+                    CEA cae(segment.Current());
+                    colorFormatMap |= static_cast<uint8_t>(cae.SupportedColorFormat());
+                    colorFormatMap |= cae.SupportedColorFormats();
+                    break;
+                }
+            }
+            colorFormatMap |= SupportedDigitalDisplayTypes();
+
+            return colorFormatMap;
+        }
+
+        uint16_t SupportedColorSpace() const {
+            uint16_t colorSpaceMap = 0;
+
+            if((_base[0x18] & (1 << 2))) {
+                colorSpaceMap |= static_cast<uint16_t>(colorspacetype::SRGB);
+            }
+
+            Iterator segment = Iterator(_segments);
+            while(segment.Next() == true) {
+                if(segment.Type() == CEA::extension_tag) {
+                    CEA cae(segment.Current());
+                    colorSpaceMap = cae.SupportedColorSpaces();
+                    break;
+                }
+            }
+
+            return colorSpaceMap;
+        }
+
+        void SupportedTimings(std::vector<uint8_t>& vicList) const {
+            Iterator segment = Iterator(_segments);
+            while(segment.Next() == true) {
+                if(segment.Type() == CEA::extension_tag) {
+                    CEA cae(segment.Current());
+                    cae.SupportedTimings(vicList);
+                    break;
+                }
+            }
+        }
+
+        uint32_t SupportedAudioFormats() const {
+            uint32_t audioFormatMap = 0;
+
+            Iterator segment = Iterator(_segments);
+            while(segment.Next() == true) {
+                if(segment.Type() == CEA::extension_tag) {
+                    CEA cae(segment.Current());
+                    audioFormatMap = cae.SupportedAudioFormats();
+                    break;
+                }
+            }
+            return audioFormatMap;
+        }
+
         void Clear() {
             _base[0] = 0x55;
             _segments.clear();
         }
 
     private:
-        inline TCHAR ManufactereChar(uint8_t value) const {
-            return static_cast<TCHAR>('A' + ((value - 1) & 0x1F));
+        inline char ManufactereChar(uint8_t value) const {
```

Comment:
See previous remark. Do not break the wchar_t/char functionality offered from Thunder. This module is by definition already depening on Thunder.

Reply:
Got it. Reverted back